docs/customer-swagger.yaml

```diff
@@ -258,6 +258,100 @@ paths:
               examples:
                 BadGatewatExample:
                   $ref: '#/components/examples/BadGatewayProblemExample'
+  /{identifier}/doiagent:
+    get:
+      summary: Get Customer doi agent with secret
+      security:
+        - CognitoUserPool: ["https://api.nva.unit.no/scopes/backend"]
```

Comment:
This is also supposed to be called from the frontend, right? I think you also need the frontend scope here.